django_components/component.py

```diff
@@ -1,5 +1,13 @@
 import warnings
 from copy import copy
+try:
+    from functools import lru_cache
+except ImportError:
+    def lru_cache(maxsize=None):  # noqa
+        def decorator(func):
+            return func
+        return decorator
+
```

Comment:
I removed support for Python 2, so this can be removed!

Reply:
You should be able to get tests to pass by rebasing your code from latest master and force pushing to your branch.